easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_libpath.patch

```diff
@@ -0,0 +1,59 @@
+Fix include path for jbig libraries
+Use CC, CFLAGS from EB.
+Drop bad make depend lines.
+
+John Dey, 2019-05-29  jfdey@fredhutch.org  fizwit@github.com
+diff -ruN jbigkit-2.1.orig/libjbig/Makefile jbigkit-2.1/libjbig/Makefile
+--- jbigkit-2.1.orig/libjbig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/libjbig/Makefile	2019-06-14 14:19:21.901910000 -0700
+@@ -1,10 +1,7 @@
+ # Unix makefile for the JBIG-KIT library
+ 
+-# Select an ANSI/ISO C compiler here, GNU gcc is recommended
+-CC = gcc
+-
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -g -O -W -Wall -ansi -pedantic # --coverage
++CFLAGS += -g -O -W -Wall -ansi -pedantic # --coverage
+ 
+ all: libjbig.a libjbig85.a tstcodec tstcodec85
+ 
+diff -ruN jbigkit-2.1.orig/Makefile jbigkit-2.1/Makefile
+--- jbigkit-2.1.orig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/Makefile	2019-06-14 14:14:58.290833000 -0700
+@@ -1,13 +1,10 @@
+ # Unix makefile for JBIG-KIT
+ 
+-# Select an ANSI/ISO C compiler here, GNU gcc is recommended
+-CC = gcc
+-
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
+-# CFLAGS = -O -g -W -Wall -Wno-unused-result -ansi -pedantic # -DDEBUG
++OPT = -O2 -W -Wno-unused-result
```

Comment:
Let's get rid of the hardcoded `-O2` so that the optimization level set by EasyBuild is used instead.